src/services/rfqm_service.ts

```diff
@@ -0,0 +1,130 @@
+import { AssetSwapperContractAddresses, MarketOperation, ProtocolFeeUtils, QuoteRequestor } from '@0x/asset-swapper';
+import { BigNumber } from '@0x/utils';
+
+import { META_TX_WORKER_REGISTRY, RFQT_REQUEST_MAX_RESPONSE_MS } from '../config';
+import { NULL_ADDRESS, ONE_MINUTE_MS } from '../constants';
+
+export interface FetchIndicativeQuoteParams {
+    apiKey: string;
+    buyAmount?: BigNumber;
+    sellAmount?: BigNumber;
+    buyToken: string;
+    sellToken: string;
+    takerAddress?: string;
+}
+
+export interface FetchIndicativeQuoteResponse {
+    buyAmount: BigNumber;
+    buyTokenAddress: string;
+    price: BigNumber;
+    sellAmount: BigNumber;
+    sellTokenAddress: string;
+    gas: BigNumber;
+    allowanceTarget?: string;
+}
+
+export const EMPTY_QUOTE_RESPONSE = {
+    buyAmount: new BigNumber(0),
+    buyTokenAddress: '',
```

Comment:
Can you simply omit invalid fields here?

Reply:
removing this obj